kolibri/plugins/media_player/assets/src/views/index.vue

```diff
@@ -106,15 +105,9 @@
       },
     },
     methods: {
-      getLangName(langCode) {
-        if (LangLookup[langCode]) {
-          return LangLookup[langCode].native_name;
-        }
-        return langCode;
-      },
       isDefaultTrack(langCode) {
         const shortLangCode = langCode.split('-')[0];
-        const shortGlobalLangCode = this.videoLang.split('-')[0];
```

Comment:
Can leave this for now, but it might be good to make this a more nuanced check in future, in the case of e.g. `es-mx` vs `es-es` subtitles.

We should also look at fleshing out the core i18n utils to handle this kind of logic (including fallbacks for checking for a long match, then a short match).